lib/debugger/clients/wdrepl.js

```diff
@@ -0,0 +1,92 @@
+var repl = require('repl');
```

Comment:
Maybe we could rename this file - it feels like we're overloading 'repl' by having it both be a client and a mode.